results/results.js

```diff
@@ -0,0 +1,109 @@
+//food app API codes
+var foodApiID = '3c019435';
+var foodApiKey = '989fd6474c5656909bd6106a3b238820';
+
+//get query selections from choice buttons
+var qword = JSON.parse(localStorage.getItem('keywords'));
```

Comment:
You might want to store the output of localStorage.getItem in a variable before trying to parse it, in case there is nothing in local storage for the key you're trying to retrieve. For example, if localStorage.getItem('keywords') returns nothing, your JSON.parse will throw an error here because it can't parse nothing.